code/game/area/areas.dm

```diff
@@ -497,8 +497,10 @@ var/list/teleportlocs = list()
 	if(!T || !isturf(T))
```

Comment:
as in, i'm not the one who added it, and I'm scared of touching something I never did, wew.